Fix http strings to https where possible

hassanelnajjar opened the pull request:
search for http: and check the url using chrome and then change the secure url to https

relates #73

## Changes

This pull request makes the following changes:

search for http: and check the url using chrome and then change the secure url to https

## Why

<!-- Why does this PR propose these changes? Take as much space as you need to explain. -->
<!-- If there are GitHub issues that this pull request addresses, please list them here. -->
For #

## Testing/Questions

Features that this PR affects:

- 

<!-- If there are no questions, please remove the questions section. -->
Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [x] Does this work for the test cases provided in https://github.com/INN/pym-shortcode/blob/master/docs/maintainer-notes.md#testing-before-release ?
- [ ] 

Steps to test this PR:

1. <!-- list any configuration changes, settings, test content, or other things necessary to test this change. -->

## Additional information

INN Member/Labs Client requesting: (if applicable)

- [x] Contributor has read INN's [GitHub code of conduct](https://github.com/INN/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] Contributor would like to be mentioned in the release notes as: (fill in this blank)
- [x] Contributor agrees to the license terms of this repository.

hassanelnajjar committed:
search for http: and check the url using chrome and then chang the secure url to https
relates #73

benlk commented:
I'm no longer a maintainer on this project, so the folks at Automattic can overrule me on this, but:

In `js/pym.js`, you do change one `http` to `https`, but you also changed the whitespace and formatting of the entire file. That's not optimal, especially since `js/pym.js` is a file that was copied in directly from a third party. Can you revert the whitespace changes, please?

hassanelnajjar committed:
fix whitespaces

hassanelnajjar committed:
fix indentation

hassanelnajjar committed:
fix indentaion

hassanelnajjar committed:
fix indentaion of pym.js

hassanelnajjar committed:
fix http to https for stackoverfolw

hassanelnajjar commented:
> I'm no longer a maintainer on this project, so the folks at Automattic can overrule me on this, but:
> 
> In `js/pym.js`, you do change one `http` to `https`, but you also changed the whitespace and formatting of the entire file. That's not optimal, especially since `js/pym.js` is a file that was copied in directly from a third party. Can you revert the whitespace changes, please?

I fixed the white spaces :)

benlk commented:
This patch looks much cleaner now; thank you!
